docs/api/api.yml

```
@@ -116,6 +116,32 @@ paths:
         410:
           description: The bag containing the item is `INACTIVE`.
 
+  /bags/filesizes/{uuid}/{path}:
```

@janvanmansum @vesaakerman is the path formatting already a definitive decision. Couple of suggestions:

1. why not `filesize` (singular) rather than `filesizes` (plural)?
2. how about swapping `filesizes` and `{uuid}`?

Basically, what I'm proposing is

```suggestion
  /bags/{uuid}/filesize/{path}:
```

---

My reasoning was that at the level `/bags/filesizes` we are still talking about all the filesizes in the bag-store, i.e. it is still a collection. With the next element `{uuid}` we pin it down to one file. Compare to the `/bags/{uuid}` endpoint.